spacy/language.py

```diff
@@ -681,6 +681,7 @@ def create_pipe(
         # interpolated variables)
         if raw_config:
             filled = filled.merge(raw_config)
+        filled["factory"] = internal_name
```

Comment:
Is there any reason for this line to be duplicated ? (already in line 674)

Reply:
Yes, the line above it overwrites the value from 674. It might make sense to just delete 674, but I'd need to check.